runtime/int.go

```diff
@@ -430,6 +441,20 @@ func intCheckedMul(v, w int) (int, bool) {
 	return x, true
 }
 
+func intCheckedPow(v, w int) (int, bool) {
+	// first try the shortcut - assuming the this operation is
```

Comment:
Is this a reasonable assumption? I have no familiarity with the relative speed of float pow vs. a multiplication loop.

Also, please make comments full sentences (capitalize and period at the end.)